.github/workflows/ci-actions.yml

```diff
@@ -95,6 +95,30 @@ jobs:
       - name: Delete Local Artifacts From Cache
         shell: bash
         run: rm -r ~/.m2/repository/io/quarkus
+      - name: Get GIB arguments
```

Comment:
I was looking long and hard for a possibility to have some kind of conditional logic in the `env` section but it seems this is not supported by GH Workflows.
That's why I ended up with this build step that creates an output that is then exposed as a job output to subsequent jobs.
I could have created a spearate job for this but then I would need to extend `needs` of each job.